nixos/modules/hardware/video/nvidia.nix

```diff
@@ -174,6 +202,14 @@ in
         assertion = !(syncCfg.enable && offloadCfg.enable);
         message = "Only one NVIDIA PRIME solution may be used at a time.";
       }
+      {
+        assertion = !(syncCfg.enable && cfg.powerManagement.finegrained);
+        message = "Sync precludes powering down the NVIDIA GPU.";
+      }
+      {
+        assertion = cfg.powerManagement.enable -> offloadCfg.enable;
```

Comment:
Should this be `cfg.powerManagement.finegrained` instead?

Reply:
#139630